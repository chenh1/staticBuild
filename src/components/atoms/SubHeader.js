import React from 'react';

const SubHeader = ({styleClass, text}) => {
    return (
        <h3 className={styleClass}>
            {text}
        </h3>
    )
};

export default SubHeader;