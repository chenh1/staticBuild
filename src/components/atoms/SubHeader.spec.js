import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import SubHeader from './SubHeader';

test('renders h3 tag with text', () => {
    const tree = renderer.create(<SubHeader
        text="Mock text"
        customClass="mock-class" />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});

test('renders h3 tag with default header-three class', () => {
    const tree = renderer.create(<SubHeader
        text="Mock text" />
    ).toJSON();

    expect(tree).toMatchSnapshot();
});