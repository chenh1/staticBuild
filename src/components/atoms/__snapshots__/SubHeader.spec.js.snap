// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders h3 tag with default header-three class 1`] = `
<h3
  className="header-three"
>
  Mock text
</h3>
`;

exports[`renders h3 tag with text 1`] = `
<h3
  className="mock-class"
>
  Mock text
</h3>
`;
